hbase-server/src/main/resources/hbase-webapps/regionserver/region.jsp

```diff
@@ -39,16 +39,22 @@
   String regionName = request.getParameter("name");
   HRegionServer rs = (HRegionServer) getServletContext().getAttribute(HRegionServer.REGIONSERVER);
   FileSystem fs = rs.getFileSystem();
-
-  HRegion region = rs.getRegion(regionName);
+  HRegion region = null;
+  if (regionName != null) {
+    region = rs.getRegion(regionName);
+  }
```

Comment:
We input directly into the browser http://localhost:16030/region.json, without specifying a region name (http://localhost:16030/region.json?name=123456789), we would get NPE in here.

Although this situation does not occur frequently, it is best to handle it.